### PR TITLE
Add the Commerce settings JS

### PR DESCRIPTION
**Pull request by wvega**
# Summary

This PRs adds the Commerce settings JS used to show a modal if the selected category changes.

## Details

The script also enables tooltips.

## UI Changes

A modal is shown if the selected category changes: https://cloud.skyver.ge/2Nuy4zkR

## Open Questions

**Should we show a modal (and use a different message) when the merchant clears the selected category?**

This PR does not show modal when the selected category is cleared because there is no value to inherit in that case and I though the current message would be confusing.

### Setup

1. Add `add_filter( 'wc_facebook_commerce_is_available', '__return_true' )`
1. Add `add_filter( 'wc_facebook_commerce_is_connected', '__return_true' )`

#### Select a different category

1. Go to WooCommerce > Facebook > Instagram Checkout
1. Select a different category
1. Save changes
    - [x] The modal is shown
1. Click Cancel
    - [x] The form is _not_ submitted
1. Save changes again
    - [x] The modal is shown
1. Click Update default Google product category
    - [x] The form is submitted
1. Wait for the page to load
    - [x] The selected category was saved

#### Clear the selected category

1. Go to WooCommerce > Facebook > Instagram Checkout
1. Clear the default category clicking the x icon on the first select field
1. Save changes
    - [x] The form is submitted 
1. Wait for the page to load 
    - [x] No category is selected

## Before merge

- [ ] I have confirmed these changes in each supported minor WooCommerce version

### CODE REVIEW

File: includes/Admin/Settings_Screens/Commerce.php
Comment (ChaseWiseman) [inline includes/Admin/Settings_Screens/Commerce.php:75]: whoops! Looks like some debug code sneaked in. The pitch also doesn't bold "Products" - was that just for testing?

**Review by ChaseWiseman — CHANGES_REQUESTED** [review]
@wvega looks good and works well! Just a bit of test code that needs to be removed

As for the open question, I've raised it [here](https://docs.google.com/document/d/1FgojSVozyO7AnLgoS-qIe1XT0pE78tYEHqnPJFhO_tY/edit?disco=AAAAKRaP73s). We can still merge this once approved, then open a new story depending on that question's outcome.

File: includes/Admin/Settings_Screens/Commerce.php
Comment (wvega) [reply includes/Admin/Settings_Screens/Commerce.php]: Thank you for catching that one @ChaseWiseman. Yes, the `<strong>` and the `<script>` tags were both for testing only. I remove them from the string in 6a1b21d4

**Review by ChaseWiseman — APPROVED** [review]

**Pull request merged**
